autolab/Autolab
# Manage Submissions Button Functionalities

Resolves #1849 

## Description
Add functionality to majority of buttons (exceptions listed below) on the redesigned Manage Submissions interface. Exceptions are:
- Missing Submissions (should still have old functionality instead of filter, this will be added in #1857)
- Change Grade Type (this button will be changed to a dropdown, details TBD, and added with #1855)

Also includes excused student marker (for #1855) and some commented code for the excused popup (for #1855, in-progress).

Includes a bug fix where only a certain number of table entries were shown— now the Manage Submissions table should show all submissions for an assignment.

## Testing
Main
- Create Submission, Download Final Submissions, and Manage Extensions should all work as intended
- Missing Submissions should work but not in the redesigned manner

Inline
- View Submission File (in row)
- Delete Submission (in row)
- [For Autograded] Regrade Submission (in row)

Selected
- Select Submissions/All should activate the Regrade/Delete/Download Selected buttons (should originally be disabled)
- [For Autograded] Regrade multiple submissions
- Delete multiple submissions
- Download multiple submissions

(Note: Original issue mentions changing wording of "Excuse Selected"/"Unexcuse Selected" in the button. This functionality was implemented in the first commit in this PR, but was removed upon further discussion in order to have a dropdown "Change Grade Type" button instead.)

## Types of changes

## Checklist:
